source/crud/tests/db/db-aggregate.yml

```diff
@@ -1,11 +1,13 @@
+database_name: &database_name "admin"
+
 data: []
 
 tests:
     -
         description: "Aggregate with $currentOp"
         operation:
             name: db-aggregate
-            database_name: admin
+            object: database
```

Comment:
I wonder if `name` should be changed to `aggregate` since we can now use `object` to differentiate. Either way, drivers that don't yet support database-level aggregation would fail on this spec test due to `db-aggregate` not being recognized or attempting to run `$currentOp` on a collection-level aggregation.

Reply:
@ShaneHarvey: thoughts?

Reply:
Yeah I like the idea of changing name to simply `aggregate`.